databases/src/connect/ConnectDatabaseAccountPromptStep.ts

```diff
@@ -0,0 +1,158 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Just verifying - all the code additions here were straight copied from the Databases Extension right? Or are there changes in addition to copying code over?

Reply:
Do you mean specifically in this file? Any under "create" or "utils" folder is directly copied over from Databases extension. Everything else including "connect" folder is new.